docs/content/preview/deploy/multi-dc/async-replication/_index.md

```diff
@@ -46,12 +46,12 @@ For information on xCluster deployment architecture, replication scenarios, and
 
 ## Best practices
 
-- Set the YB-TServer [cdc_wal_retention_time_secs](../../../reference/configuration/all-flags-yb-tserver/#cdc-wal-retention-time-secs) flag to 86400 on both source and target universe.
+- Set the YB-TServer [cdc_wal_retention_time_secs](../../../reference/configuration/all-flags-yb-tserver/#cdc-wal-retention-time-secs) flag to 86400 on both source and target.
 
-    This flag determines the duration for which WAL is retained on the source universe in case of a network partition or a complete outage of the target universe. The value depends on how long a network partition of the source universe or an outage of the target universe can be tolerated.
+    This flag determines the duration for which WAL is retained on the source in case of a network partition or a complete outage of the target. For xCluster Replication, you should set the flag to a value greater than the default. The goal is to retain write-ahead logs (WALs) during a network partition or target outage until replication can be restarted. Setting this to 86400 (24 hours) is a good rule of thumb, but you should also consider how quickly you will be able to recover from a network partition or target outage.
 
 - Make sure all YB-Master and YB-TServer flags are set to the same value on both the source and target universes.
 
 - Monitor CPU usage and ensure it remains below 65%. Note that xCluster replication typically incurs a 20% CPU overhead.
```

Comment:
Monitor CPU usage and ensure it remains under 65%. Note that xCluster replication typically incurs a 20% CPU overhead.